src/views/Ifos/index.tsx

```diff
@@ -2,15 +2,19 @@ import { SubMenuItems } from '@pancakeswap/uikit'
 import { useTranslation } from 'contexts/Localization'
 import { PageMeta } from 'components/Layout/Page'
 import { useRouter } from 'next/router'
+import { usePoolsPageFetch, useFetchIfo } from 'state/pools/hooks'
 import Hero from './components/Hero'
+import IfoProvider from './contexts/IfoContext'
 
 export const IfoPageLayout = ({ children }) => {
   const { t } = useTranslation()
   const router = useRouter()
   const isExact = router.route === '/ifo'
+  usePoolsPageFetch()
```

Comment:
Is this too much to fetch all pools for ifo page?

Reply:
@chef-jojo  Updated.